Tools/build/deepfreeze.py

```diff
@@ -461,6 +465,7 @@ def generate(args: list[str], output: TextIO) -> None:
             with printer.block(f"if ({p} < 0)"):
                 printer.write("return -1;")
         printer.write("return 0;")
+    printer.write(f"\nuint32_t _Py_next_func_version = {next_code_version};\n")
```

Comment:
does it need to be incremented here?

Reply:
No, as it is the next version. It gets incremented when used.